File: VideoServer-Play-Scala-Slick-NoAuth/app/videosvc/controllers/WebApplication.scala
package videosvc.controllers

import play.api.mvc._

class WebApplication extends Controller {

/*
  def index = Action {
    Ok(videosvc.views.html.index("Your new application is ready."))
  }
*/

  def index = Action {
    Redirect("/videoApp")
  }

  def app = TODO

//  def app = Action {
//    Ok(videosvc.views.html.index("Your new application is ready."))
//  }

}

File: VideoServer-Play-Java-Ebean-NoAuth/app/videosvc/controllers/WebApplication.java
package videosvc.controllers;

import play.Logger;
import play.mvc.Controller;
import play.mvc.Result;
import videosvc.views.html.index;

public class WebApplication extends Controller {

    private final Logger.ALogger l = Logger.of(getClass());


    public Result index() {

        l.debug("index()");

        return ok(index.render("Your new application is ready."));
    }
}
